enforcer-parent/enforcer/pom.xml

```diff
@@ -634,4 +637,11 @@
         </dependency>
     </dependencies>
 
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <carbon.apimgt.common.analytics.version>9.27.17-SNAPSHOT</carbon.apimgt.common.analytics.version>
+        <org.wso2.am.analytics.publisher.version>1.1.1-SNAPSHOT</org.wso2.am.analytics.publisher.version>
```

Comment:
Please remove the versions from here as versioning will be fatched form the parent pom. Also always use released versions. Not SNAPSHOT version